orchagent/vnetorch.cpp

```diff
@@ -1967,6 +2221,219 @@ void VNetRouteOrch::updateVnetTunnel(const BfdUpdate& update)
     }
 }
 
+void VNetRouteOrch::updateVnetTunnelCustomMonitor(const MonitorUpdate& update)
```

Comment:
Please add comments/description to this function

Reply:
done